content/maxwell1_fundamentals/dipole_sources_in_homogeneous_media/magnetic_dipole_frequency/fields.rst

```diff
@@ -3,8 +3,90 @@
+
+	- In this case, is there a non-zero electric field and/or current density?
+	- Does the magnetic field have any radial components?
+	- Does the magnetic field have any quadrature (imaginary) components?
+	- Do any aspects of the magnetic field change when you alter the physical properties of the medium?
+	- Do your observations make sense when considering the :ref:`DC approximation<frequency_domain_magnetic_dipole_asymptotics_DC>`?
+
+**Near-Field Approximation:**
+
+Now set the conductivity of the background to 1 S/m and set the frequency to 1 Hz. According to our asymptotic approximations, the :ref:`near-field<frequency_domain_magnetic_dipole_asymptotics_near>` (:math:`| kr | \ll 1`) the magnetic field should behave like the DC case and the electric field should have only quadrature (imaginary) components.
+
+	- Examine the electric and magnetic fields. Do your observations compare well to what your would expect?
+	- Increase the frequency by a factor of 10. Does the strength of the magnetic field change? Does the strength of the electric field change? Does this make sense when considering the near-field approximations?
+	- Are the induced electric fields rotational about the dipole?
+
+Now slowly increase the frequency by factors of 10. When you reach 1000 Hz, notice that at sufficient distance from the dipole, the near-field approximation is no longer valid. However near the dipole source, the fields more or less behave like DC fields.
+
+	- At what distance is the near-field approximation no longer valid at 1000 Hz? Use this value to confirm that :math:`| kr | \ll 1`.
+
+**The Inductive Response:**
+
+According to `Faraday's law<faraday>`, the effects of EM induction increase as frequency increases. Set the conductivity to 0.1 S/m and choose a point (x,y,z) = (40m, 0m, 0m). Examine the x,y and z components of the electric and magnetic fields.
+
+	- At what frequency do the effects of EM induction become significant?
+	- Now increase the background conductivity to 1 S/m and examine the same location. At what frequency do the effects of EM induction become significant?
+	- Now choose a location closer to the dipole source (x,y,z) = (10m, 0m, 0m). At what frequency do the effects of EM induction become significant compared to the primary field?
+
+**Magnetic Permeability and Dielectric Permittivity**
+
+Set the log-conductivity to .
+
+
+	- Try increasing the relative permeability (:math:`\mu_r`). Do you notice any significant changes in the shape and amplitude of the electric and magnetic fields?
+	- Now try increasing the relative permittivity (:math:`\varepsilon_r`). When you do this at low frequencies, do you notice any significant changes in the shape and amplitude of the electric and magnetic fields? How about when you do this at high frequencies?
+
+
+**Hypothetical Scenario 1:**
+
+*I put this here in case we wanted to make a hypthetical scenario where these equations could be used to solve a practical problem.*
```

Comment:
cool this is a great idea. I am not sure if we want to leave the suggestion live on the web though, maybe just comment it out but leave it in the text? and even create an issue if you like? that way we can have a conversation about it.